### DISCUSSION TITLE
Browser back button handling to close dialogs

### DISCUSSION THREAD

**Discussion by CheIIau** [quasarframework/quasar#14283] [CLI - PWA mode]
Hello, I think it will be useful if quasar had a feature to close dialogs and popups on back button in browser. For now i have my own logic in components to do this, but it seems quite ugly. 
Is there a way to add some directive on dialog like 'v-back-button-close' or maybe some other implementation of this like $q.dialog({ back:true, ... }) or some global flag for this in quasar.config? I see it like this: we check if there any open dialogs in app, and if so, prevent navigation and close them. if there none - navigate to previous route.
Or this feature cannot be implemented and quasar can't handle user navigation like this?
P.S. I know 3 years ago user asked a similar question, but there was no answer 🤷

**Comment by drogerie21** [comment]
need the same functionality!
